test/integration/stats_integration_test.cc

```diff
@@ -213,8 +213,9 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithStats) {
   // 2019/05/07  6794     49957               Stats for excluded hosts in cluster
   // 2019/04/27  6733     50213               Use SymbolTable API for HTTP codes
   // 2019/05/31  6866     50157               libstdc++ upgrade in CI
+  // 2019/06/03  7146     49137               absl update
 
-  EXPECT_EQ(m_per_cluster, 50157);
+  EXPECT_EQ(m_per_cluster, 49137);
```

Comment:
This is a significant improvement. I'm surprised the absl version makes this much difference, and I am guessing this is not due to the change to node_hash_map as that's not per-cluster.